Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -72,6 +72,10 @@ - (void)enforceTextAttributesIfNeeded
   if ([textAttributes valueForKey:NSForegroundColorAttributeName] == nil) {
       [textAttributes setValue:[UIColor blackColor] forKey:NSForegroundColorAttributeName];
   }
+  // We need to add check here.See https://github.com/facebook/react-native/issues/28488
```

Comment:
We need to move that to the `defaultTextAttributes` setter of the concrete class of `backedTextInputView`.

Reply:
Concretely to the `RCTUITextField`.

Reply:
> Concretely to the `RCTUITextField`.

Done. :)